platformio_esp32_envs.ini

```diff
@@ -187,6 +187,7 @@ board                     = esp32_4M
 lib_deps                  = ${esp32_common.lib_deps}, ServoESP32
 build_flags               = ${esp32_common.build_flags}  
                             -DFEATURE_ARDUINO_OTA
+                            -DPLUGIN_DISPLAY_COLLECTION
```

Comment:
This fixes the ESP32 `DISPLAY` build to again include the 'Display collection' plugins.